velocity-transition-group.js

```diff
@@ -281,27 +283,39 @@ var VelocityTransitionGroup = React.createClass({
     // animations) so that we can then animate out. Velocity typically makes these transitions
     // very smooth, correctly animating from whatever state the element is currently in.
     if (entering) {
-      completeFn();
-      completeFn = null;
+      doneFn();
+      doneFn = null;
     } else {
       Velocity(nodes, 'stop');
     }
 
+    var combinedCompleteFn;
+    if (doneFn && opts.complete) {
+      var optsCompleteFn = opts.complete;
+      combinedCompleteFn = function () {
+        doneFn();
+        optsCompleteFn();
+      };
+    } else {
+      // One or the other or neither.
```

Comment:
Wanted to make "or neither" clear, that we're ok with this being undefined if there are no callbacks desired.